### PR TITLE
Preserve line breaks and format version information

### PR DESCRIPTION
**Pull request by esanzgar**
I have tested successfully on the latest version of Chrome, Safari and
Firefox. I also tested successfully on an Android and iOS devices.

I couldn't tested more exhaustively on other desktop/mobile browser
versions because SauceLabs didn't allowed me to retrieve the pasted
text.

Closes https://github.com/hypothesis/product-backlog/issues/1180

### CODE REVIEW

**Comment by lyzadanger** [comment]
@esanzgar FYI, the bug referenced here will be _partially_ fixed by this PR but not entirely.

File: src/sidebar/helpers/version-data.js [hunk 1]
```diff
@@ -35,15 +35,15 @@ export default class VersionData {
       }
     }
 
-    this.timestamp = new Date().toString();
+    this.version = '__VERSION__'; // replaced by versionify
```
Comment (esanzgar) [inline src/sidebar/helpers/version-data.js:38]: I just reordered that the properties appears in the same order both in the UI and code.

File: src/sidebar/helpers/version-data.js [hunk 2]
```diff
@@ -53,13 +53,13 @@ export default class VersionData {
...
    */
   asFormattedString() {
-    let versionString = '';
-    for (let prop in this) {
-      if (Object.prototype.hasOwnProperty.call(this, prop)) {
-        versionString += `${prop}: ${this[prop]}\r\n`;
-      }
-    }
-    return versionString;
+    return `Version: ${this.version}
```
Comment (esanzgar) [inline src/sidebar/helpers/version-data.js:56]: The `\r` (return) character is not necessary. I tested in Microsoft 10.

**Review by esanzgar — COMMENTED** [review]
I added a second commit to address the order in the version data.

File: src/sidebar/util/copy-to-clipboard.js [hunk 3]
```diff
@@ -10,7 +10,7 @@
  * @param {string} text
  */
 export function copyText(text) {
-  const temp = document.createElement('input');
+  const temp = document.createElement('textarea'); // use textarea instead of input to preserve line breaks
```
Comment (lyzadanger) [inline src/sidebar/util/copy-to-clipboard.js:13]: I'm glad you added a comment here; I was going to suggest it.

File: src/sidebar/helpers/version-data.js (hunk 2 above)
Comment (lyzadanger) [inline src/sidebar/helpers/version-data.js:56]: Thanks for refactoring this. I think I'm guilty of trying to make it overly-generic in its first iteration.

**Review by lyzadanger — APPROVED** [review]
This fixes both problems, thanks! The commits look good, but could you please update the title and description of the PR before merging for clearer history later?

**Comment by lyzadanger** [comment]
p.s. I tested quickly in Mac OS Chrome, Safari and FF. I'm taking your word for it on the mobile browsers here.

**Pull request merged**